Update Name Servers How-to articles (#1943)
dnsimple/dnsimple-support: content/articles/delegating-dnsimple-hosted.md

---
title: Delegating a Domain registered with another Registrar to DNSimple
excerpt: Point your domain's name servers to DNSimple when the domain is registered at another registrar.

> meta: Delegate a domain that is registered at another registrar to DNSimple by updating name servers at that registrar to the DNSimple name server hostnames listed in DNSimple Name Servers.

Reviewer: ```suggestion
meta: Delegate a domain registered at another registrar to DNSimple by updating the name servers at your registrar to the DNSimple name server hostnames shown on the DNSimple Name Servers page.
```
